src/core/api/chart_repository.go

```diff
@@ -87,7 +88,13 @@ func (cra *ChartRepositoryAPI) Prepare() {
 // GetHealthStatus handles GET /api/chartrepo/health
 func (cra *ChartRepositoryAPI) GetHealthStatus() {
 	// Check access
-	if !cra.requireAccess(cra.namespace, accessLevelSystem) {
+	if !cra.SecurityCtx.IsSysAdmin() {
+		if !cra.SecurityCtx.IsAuthenticated() {
```

Comment:
`if !cra.SecurityCtx.IsAuthenticated() `should be moved outside  `!cra.SecurityCtx.IsSysAdmin() `

Reply:
it be moved outside